chalicelib/app_utils.py

```diff
@@ -179,13 +180,15 @@ def get_favicon(server):
     Tries to grab favicon from the given server. If it's not found it will use
     the default favicon (data)
     """
-    try:
-        favicon = server + 'static/img/favicon-fs.ico'
+    def favicon_is_valid(favicon):
         res = requests.head(favicon)
-        assert res.status_code == 200
-    except:
-        favicon = DEFAULT_FAVICON
-    return favicon
+        return res.status_code == 200
+
+    if 'cgap' in server:
```

Comment:
There's a better way to do this if you took the bs_env as an argument, using `dcicutils.env_utils.is_cgap_env`. But even if not, we should maybe have a similar `is_cgap_server(server)` or even `get_favicon_url(bs_env)` rather than do string operations here.